### ISSUE TITLE
Removing included files

### ISSUE THREAD

**Issue by elmarti** [davidd8/meteor-external-file-loader#10]
I've used this library to include Reveal.JS because it corrupts my css if I use the Meteor package, however what i'm finding now is that when I leave the Reveal.js template where I've used EFL, the files are still included. Is there a way to explicitly remove them?

**Comment by m-pokrovskii** [comment]
+1

**Comment by elmarti** [comment]
Hey m-pokrovskii in the end I simply inserted all of the Reveal library into the onCreated and then assigned the Reveal object to window.Reveal, which solved the problem for me. I have since found a way to load files into a particular context, but I think that might do better on a different project.

**Comment by m-pokrovskii** [comment]
Thanks, but my issue is quite different: I load a css files, and i don't want them on the anothers pages.

**Comment by elmarti** [comment]
This is another thing that I have used to separate the frontend css from the backend dashboard css -in this, my front.css is in the public folder. 
`Template.parent_template.onRendered(function() {
    let fileref = document.createElement("link");
    fileref.setAttribute("rel", "stylesheet");
    fileref.setAttribute("type", "text/css");
    fileref.setAttribute("href", "/front.css");
});`

**Comment by elmarti** [comment]
I don't think this is being developed any more so my made my own for just CSS. This does that and allows you to explicitly remove them https://atmospherejs.com/elmarti/super-css-loader

**Comment by m-pokrovskii** [comment]
Thanks for this link
